### ISSUE TITLE
`require` in ESM build

### ISSUE THREAD

**Issue by segevfiner** [Giphy/giphy-js#381]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

The ESM `index.js` of this package includes a call to `require`, which fails to bundle for the browser properly:
```js
var _a;
import { appendGiphySDKRequestHeader, getGiphySDKRequestHeaders } from '@giphy/js-util';
export { default as GiphyFetch } from './api';
export { serverUrl, setServerUrl } from './constants';
export { default as FetchError } from './fetch-error';
export * from './option-types';
export { gifPaginator } from './paginator';
export { default as request } from './request';
export * from './result-types';
var version = require('../package.json').version;
// since we have multiple SDKs, we're defining a hieracrchy
// Fetch API is lowest
if (!((_a = getGiphySDKRequestHeaders()) === null || _a === void 0 ? void 0 : _a.get("X-GIPHY-SDK-NAME"))) {
    // send headers with library type and version
    appendGiphySDKRequestHeader("X-GIPHY-SDK-NAME", 'FetchAPI');
    appendGiphySDKRequestHeader("X-GIPHY-SDK-VERSION", version);
}
//# sourceMappingURL=index.js.map
```

Note: `var version = require('../package.json').version;` which is invalid in ESM.

### Expected behavior 🤔

ESM is ESM. It doesn't have require.

### Steps to reproduce 🕹

Steps:

1. Import the package in an Vite project.

### Platform 🌍

- OS: macOS 13.4
- Browser: Google Chrome 114.0.5735.106

### GIPHY-JS SDK version

@giphy/js-fetch-api 4.9.2

### TypeScript version

v5.0.4

### Additional context 🔦

Doesn't happen in v4.8.1

**Comment by giannif** [comment]
@segevfiner I have a fix for this in a branch which will go out today

**Comment by giannif** [comment]
fixed in @giphy/js-fetch-api: 4.9.3
